src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenClosureLambdaTests.cs

```diff
@@ -4244,7 +4244,7 @@ static void Main()
         }
 
         [WorkItem(1019237, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1019237")]
-        [Fact]
+        [ConditionalFact(typeof(ClrOnly), Reason = "https://github.com/mono/mono/issues/10838")]
```

Comment:
> "https://github.com/mono/mono/issues/10838")] [](start = 50, length = 46)

I don't think it's a good idea to link to a mono repo issue.
When that issue gets resolved, they will not think to come to rolsyn and uncondition this test.

Reply:
How will linking to a Roslyn issue improve it though? The same argument applies. If they resolve the bug and don't tell us then the Roslyn issue remains active and the test is disabled.  This is something I've gone back and forth on. I haven't come up with a great answer here. 

What I've been doing for this set of changes is:
1. File a Mono bug and link to that
2. Have an uber bug where I can track all Mono issues we currently depend on. That way I can quickly spot check if anything got resolved that we depend on. [https://github.com/dotnet/roslyn/issues/29859](https://github.com/dotnet/roslyn/issues/29859)



---
In reply to: [221104583](https://github.com/dotnet/roslyn/pull/30170#discussion_r221104583) [](ancestors = 221104583)

Reply:
In roslyn issues, I try to leave a note as a reminder to check for references in the code. 
That said, if the issue is in mono, we'd still have to remember to check the roslyn issue when the mono issue gets closed...
Fine to leave on mono side. Maybe add a note to ping us (nonzero likelihood we'll get notified ;-) ).